tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -158,7 +158,112 @@ Verify "Notebook Images Are Building" Is Not Shown When No Images Are Building
     Wait Until All Builds Are Complete  namespace=redhat-ods-applications
     RHODS Notification Drawer Should Not Contain  message=Notebooks images are building
 
+Verify Favorite Resource Cards
+    [Tags]    ODS-389
+    ...       Tier1
+    [Documentation]    Veerifies the item in Resource page can be marked se favourite.
+    ...                It checks if favorite items are always listed as first regardless
+    ...                the view type or sorting
+    Click Link    Resources
+    Sleep    5s
```

Comment:
maybe you can use from all your sleep
[Wait Until Element Is Visible](https://robotframework.org/SeleniumLibrary/SeleniumLibrary.html#Wait%20Until%20Element%20Is%20Visible)

Reply:
Done